### PR TITLE
Refactored to use an object oriented approach

### PR DESCRIPTION
**Pull request by London804**

### CODE REVIEW

File: memory.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
+$(function() {
+	var lastClicked;
+
+	$('td').click(function(){
+		if (lastClicked === undefined){
```
Comment (afeld) [inline memory.js:5]: Unless you need to check for `undefined` specifically, I would just simplify this to `if (!lastClicked)`.

File: memory.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+			// first click
+			lastClicked = this;
+		} else if (this != lastClicked && $(lastClicked).text() === $(this).text()){
```
Comment (afeld) [inline memory.js:8]: How would you explain what this condition is checking? Try moving it to a method to make it more descriptive.

**Comment by afeld** [comment]
Good first pass – would like to see it refactored to use objects. Thanks!

**Comment by London804** [comment]
Hey Aidan, I did refactor it to use objects. Look at this memory00.js to see!

File: index.html [hunk 3]
```diff
@@ -2,7 +2,50 @@
...
+        <td>A</td>
+      </tr>
+      <tr>
+        <td>C</td>
+        <td>B</td>
+      </tr>
+      <tr>
+        <td>B</td>
+        <td>C</td>
+      </tr>
```
Comment (afeld) [inline index.html:45]: Makes it a lot easier to play if I can see the values of all the tiles :smirk: 

screen shot 2015-11-17 at 8 06 15 am

**Comment by afeld** [comment]
Indeed you did – sorry, not sure how I missed that! Updated your score.

**Pull request closed without merging**